packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -329,7 +329,7 @@ const useDropdown = (): UseDropdownReturnValue => {
 
       const filteredIndexes = filteredValues
         .map((filteredValue) => options.findIndex((option) => option.value === filteredValue))
-        .sort();
+        .sort((a, b) => a - b);
```

Comment:
why?

Reply:
It is expected to sort indexes which are numbers and js does weird things with numbers - 
<img width="318" alt="image" src="https://github.com/razorpay/blade/assets/30949385/e521018f-a3ff-488d-897c-a5456efcf16a">


So in AutoComplete, when we add more than 10 items in search, the focus was jumping incorrectly to 10th item after 1st